source/uwp/Renderer/lib/Util.cpp

```diff
@@ -73,6 +76,199 @@ bool Boolify(const boolean value)
+                break;
+            case ABI::AdaptiveCards::Uwp::ElementType::DateInput:
+                baseCardElement = GetSharedModel<AdaptiveCards::BaseCardElement, AdaptiveCards::DateInput, ABI::AdaptiveCards::Uwp::IAdaptiveCardElement, AdaptiveCards::Uwp::AdaptiveDateInput>(item);
+                break;
+            case ABI::AdaptiveCards::Uwp::ElementType::FactSet:
+                baseCardElement = GetSharedModel<AdaptiveCards::BaseCardElement, AdaptiveCards::FactSet, ABI::AdaptiveCards::Uwp::IAdaptiveCardElement, AdaptiveCards::Uwp::AdaptiveFactSet>(item);
+                break;
+            case ABI::AdaptiveCards::Uwp::ElementType::Image:
+                baseCardElement = GetSharedModel<AdaptiveCards::BaseCardElement, AdaptiveCards::Image, ABI::AdaptiveCards::Uwp::IAdaptiveCardElement, AdaptiveCards::Uwp::AdaptiveImage>(item);
+                break;
+            case ABI::AdaptiveCards::Uwp::ElementType::ImageSet:
+                baseCardElement = GetSharedModel<AdaptiveCards::BaseCardElement, AdaptiveCards::ImageSet, ABI::AdaptiveCards::Uwp::IAdaptiveCardElement, AdaptiveCards::Uwp::AdaptiveImageSet>(item);
+                break;
+            case ABI::AdaptiveCards::Uwp::ElementType::NumberInput:
+                baseCardElement = GetSharedModel<AdaptiveCards::BaseCardElement, AdaptiveCards::NumberInput, ABI::AdaptiveCards::Uwp::IAdaptiveCardElement, AdaptiveCards::Uwp::AdaptiveNumberInput>(item);
+                break;
+            case ABI::AdaptiveCards::Uwp::ElementType::TextBlock:
+                baseCardElement = GetSharedModel<AdaptiveCards::BaseCardElement, AdaptiveCards::TextBlock, ABI::AdaptiveCards::Uwp::IAdaptiveCardElement, AdaptiveCards::Uwp::AdaptiveTextBlock>(item);
+                break;
+            case ABI::AdaptiveCards::Uwp::ElementType::TextInput:
+                baseCardElement = GetSharedModel<AdaptiveCards::BaseCardElement, AdaptiveCards::TextInput, ABI::AdaptiveCards::Uwp::IAdaptiveCardElement, AdaptiveCards::Uwp::AdaptiveTextInput>(item);
+                break;
+            case ABI::AdaptiveCards::Uwp::ElementType::TimeInput:
+                baseCardElement = GetSharedModel<AdaptiveCards::BaseCardElement, AdaptiveCards::TimeInput, ABI::AdaptiveCards::Uwp::IAdaptiveCardElement, AdaptiveCards::Uwp::AdaptiveTimeInput>(item);
+                break;
+            case ABI::AdaptiveCards::Uwp::ElementType::ToggleInput:
+                baseCardElement = GetSharedModel<AdaptiveCards::BaseCardElement, AdaptiveCards::ToggleInput, ABI::AdaptiveCards::Uwp::IAdaptiveCardElement, AdaptiveCards::Uwp::AdaptiveToggleInput>(item);
+                break;
+            case ABI::AdaptiveCards::Uwp::ElementType::Custom:
+                baseCardElement = std::make_shared<CustomElementWrapper>(item);
+                break;
+        }
+        containedElements.push_back(baseCardElement);
```

Comment:
>push_back [](start = 26, length = 9)

GetSharedModel may return nullptr. Do we want to add those to the containedElements anyways?